src/seat.c

```diff
@@ -189,6 +189,12 @@ configure_libinput(struct wlr_input_device *wlr_input_device)
 		wlr_log(WLR_INFO, "dwt configured");
 		libinput_device_config_dwt_set_enabled(libinput_dev, dc->dwt);
 	}
+	if (libinput_device_config_click_set_method(libinput_dev,
```

Comment:
Do a check here to avoid configuring when `dc->click_method < 0`

Reply:
basically done.